spec/amp-versioning-policy.md

```diff
@@ -1,61 +1,69 @@
+# AMP versioning and deprecations policy
 
 ## Goals
 
 - Reduce overall JS URL entropy. AMP aims to provide as few versions of the JS library and extensions as possible to improve cache hit rates for these requests.
 - Avoid breaking existing pages.
-- Keep all users of AMP on secure versions of the library.
+- Keep all users of AMP on secure and updated versions of the library.
+- Enable all AMP developers to keep their AMP pages up-to-date.
+
+## Versioning
+
+- The AMP JS library and its extensions are versioned. There is also an **[AMP release](https://github.com/ampproject/amphtml/releases/)** that corresponds to the precise code. Each major and minor version may correspond to many AMP releases.
+- AMP is **released** about once per week and each release has a unique version number. With each release, AMP pages are rendered with the latest code for the respective major/minor versions of the AMP JS library and extensions.
+
+### AMP JS library
+
+The AMP JS library is distributed via its major version only. The major version is expected to change rarely if ever (on the order of less than once per year).
+
+### AMP extensions
 
-## Non-breaking changes
+Each AMP extension is versioned independently. Publishers of AMP documents can choose the major and minor version of an extension; they cannot choose the patch version. 
 
-- If a change is backward compatible, no version should be changed.
-- This means that, for now, there is no known case under which the minor version of an extension would be changed.
+- Minor versions may be changed for breaking behavior changes *without* breaking API changes.
+- Major versions *must* be changed for any breaking API changes.
 
 ## Breaking changes
 
-Breaking changes are:
+Breaking changes must be implemented in a new version. Breaking changes are defined as:
 
-- Changes to the semantics of supported attributes and other exposed APIs.
+- Changes to exposed API semantics, e.g. supported attributes in an AMP extension.
 - Changes to CSS included in AMP that are not backward compatible.
 - Changes to the DOM structure (elements, attributes and attribute values) generated by AMP extensions that are not backward compatible.
 
 The following special cases are not considered breaking changes:
 
 - Changes to elements and their children including their CSS if the element name starts with `i-`.
-- Changes to attribute and class names starting with `-`. But this doesn't mean, CSS backward compatibility may be broken with this mechanism, where it would otherwise be considered a breaking change.
+- Changes to attribute and class names starting with `i-` (but this doesn't mean that CSS backward compatibility may be broken with this mechanism, which would be considered a breaking change).
 - Changes required to maintain the security of AMP pages.
```

Comment:
Security changes can be breaking by the definition above.  I think this should come out of here and a note added that says something to the effect of "Changes required to maintain the security of AMP pages may not follow this policy if doing so would be detrimental to security"

Reply:
Moved the security point to its own subsection.